### PR TITLE
Simplify resource ids

### PR DESCRIPTION
**Pull request by mhutch**
Simplify resource id definitions using a definition group.

### CODE REVIEW

File: main/msbuild/MonoDevelop.AfterCommon.props [hunk 1]
```diff
@@ -29,4 +29,10 @@
 		<AssemblyName>$(MSBuildProjectName)</AssemblyName>
 		<RootNamespace>$(MSBuildProjectName)</RootNamespace>
 	</PropertyGroup>
+
+	<ItemDefinitionGroup Condition="'$(ShortEmbeddedResourceIDs)'!='False'">
+		<EmbeddedResource>
```
Comment (Therzok) [inline main/msbuild/MonoDevelop.AfterCommon.props]: Nice!

What about cases where the subtype is set?

Comment (mhutch) [reply main/msbuild/MonoDevelop.AfterCommon.props]: That should still work! This just adds metadata.

Also, I'm not sure what exactly the subtype is for - VS adds it in some cases, but it isn't used by MSBuild at all.

File: main/external/fsharpbinding/MonoDevelop.FSharp.Gui/MonoDevelop.FSharp.Gui.csproj [hunk 2]
```diff
@@ -45,10 +45,7 @@
     <Reference Include="glib-sharp, Version=2.12.0.0, Culture=neutral, PublicKeyToken=35e10195dab3c99f" />
   </ItemGroup>
   <ItemGroup>
-    <EmbeddedResource Include="gtk-gui\gui.stetic">
-      <LogicalName>gui.stetic</LogicalName>
-      <SubType>Designer</SubType>
```
Comment (Therzok) [inline main/external/fsharpbinding/MonoDevelop.FSharp.Gui/MonoDevelop.FSharp.Gui.csproj]: @mhutch you deleted subtype here.

Comment (Therzok) [reply main/external/fsharpbinding/MonoDevelop.FSharp.Gui/MonoDevelop.FSharp.Gui.csproj]: I guess it's used as a viewer hint?

Comment (mhutch) [reply main/external/fsharpbinding/MonoDevelop.FSharp.Gui/MonoDevelop.FSharp.Gui.csproj]: It's back :)

**Comment by slluis** [comment]
I'm not sure VSMac can handle this. Does roundtripping work?
Also, VSMac will add the resource name when new resources are added, unless we change the naming policy.

**Comment by mhutch** [comment]
@slluis I confirmed roundtripping works, and update the policy

**Pull request merged**